exchange/exchange_test.go

```diff
@@ -568,6 +569,61 @@ func newExtRequestNoBrandCat() openrtb_ext.ExtRequest {
 	}
 }
 
+func newExtRequestTranslateCatNil() openrtb_ext.ExtRequest {
```

Comment:
Please move this helper method below the TestCategoryMappingTranslateCategoriesNil (the first place it's called) for better readability. Consider accepting a TranslateCategories argument so you don't need two helper methods.

Reply:
Moved this function to be below the first test that uses it. Also updated it to take a `translateCategories` argument so that there is only one helper function for both tests.